src/controllers/agent.js

```diff
@@ -0,0 +1,66 @@
+import dotenv from 'dotenv';
+import Model from '../../../Desktop/property-pro-api/src/models/model';
+
+const jwt = require('jsonwebtoken');
+const bcrypt = require('bcrypt');
+
```

Comment:
I suggest you use es6 through out for consistency. Instead on using import and requie together at the top level